packages/next-app/src/app/metadata/[network]/pool/[poolId]/(components)/MetadataAttributesTable/Actions.tsx

```diff
@@ -41,7 +44,7 @@ export function Actions({
           <span tabIndex={0}>
             <Dialog title={"Add attribute"} content={<PoolMetadataItemForm />}>
               <Button
-                className="bg-indigo-500 text-white hover:bg-indigo-400 focus-visible:outline-indigo-500 disabled:bg-indigo-400"
+                className="bg-indigo-500 text-white hover:bg-indigo-400 focus-visible:outline-indigo-500 disabled:bg-indigo-400 disabled:opacity-40"
```

Comment:
You can probably leverage the updated Button component in `main` to avoid having to define all these classes here.